licenses.d2iq.yaml

```diff
@@ -9,605 +9,560 @@ ignore:
   - docker.io/library/busybox:1
   - gcr.io/kubecost1/cost-model:prod-1.104.0
   - gcr.io/kubecost1/frontend:prod-1.104.0
-
-  # Fossa doesn't support scanning arbitrary C-files
   - docker.io/mesosphere/pause-busybox:3.6
   - docker.io/bitnami/bitnami-shell:11-debian-11-r106
   - docker.io/bitnami/postgres-exporter:0.12.0-debian-11-r77
   - docker.io/mesosphere/trivy-bundles:0.42.1-20230726T060317Z
   - gcr.io/google_containers/pause:3.2
-
-  # Fossa cannot scan and parse gemfile provided by the fluend project
-  # - container_image: ghcr.io/banzaicloud/fluentd:v1.14.6-alpine-5
-  #   sources:
-  #     - url: https://github.com/fluent/fluentd
-  #       ref: ${image_tag%-alpine-5}
-  #       license_path: LICENSE
   - ghcr.io/banzaicloud/fluentd:v1.14.6-alpine-5
-
-  # Fossa cannot scan C/CPP projects
-  # See: https://docs.fossa.com/docs/c-cpp
-  # - container_image: nvcr.io/nvidia/cloud-native/dcgm:${image_tag}
-  #   sources:
-  #     - url: https://github.com/NVIDIA/DCGM
-  #       ref: ${image_tag}
-  #       license_path: LICENSE
   - nvcr.io/nvidia/cloud-native/dcgm:3.1.7-1-ubuntu20.04
-
-  # Incorrect mapping was provided
-  # - container_image: docker.io/bitnami/kubectl:1.19.7
-  #   sources:
-  #     - url: https://github.com/bitnami/bitnami-docker-kubectl
-  #       ref: ${image_tag}-debian-10-r31
-  #       license_path: LICENSE
   - docker.io/bitnami/kubectl:1.19.7
   - docker.io/bitnami/kubectl:1.22.0
   - docker.io/bitnami/kubectl:1.24.1
   - docker.io/bitnami/kubectl:1.26.4
-
-  # Fossa cannot scan C/CPP projects
-  # See: https://docs.fossa.com/docs/c-cpp
-  # - container_image: docker.io/curlimages/curl:7.83.1
-  #   sources:
-  #     - url: https://github.com/curl/curl
-  #       ref: curl-${image_tag//./_}
-  #       license_path: COPYING
   - docker.io/curlimages/curl:7.83.1
-
-  # This is added as part of `kubetunnel` project, but we only use cURL from
-  # this project.
-  # See: https://github.com/mesosphere/kubetunnel/blob/v0.0.15/pkg/controllers/constants.go
   - docker.io/library/centos:8
-
-  # This container is part of nvidia extension but the source code of this
-  # repo cannot be scanned by Fossa.
-  # - container_image: nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
-  #   sources:
-  #     - url: https://github.com/NVIDIA/cuda-samples
-  #       ref: v${image_tag#vectoradd-cuda}
-  #       license_path: LICENSE
   - nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
-
 resources:
   - container_image: cr.fluentbit.io/fluent/fluent-bit:2.1.4
     sources:
-      - license_path: LICENSE
+      - url: https://github.com/fluent/fluent-bit
```

Comment:
it looks like all of this diff is just reordering keys?

Reply:
meh. I don't know why sometimes the workflow reorders everything / removes comment. the image is also bumped here but I think I will just revert the automated commit and manually bump that image to avoid changing up the whole file

Reply:
yeah i suppose eventually we'll have to update this file with the automation but i was just checking... or update the script to not reorder/remove comments 🤔

Reply:
I will ask @sonofgibs to look into when she gets back :)